main.py

```diff
@@ -42,13 +42,13 @@ def __mul__(self,other):
     
     def __gt__(self,other):
         if isinstance(other,Distance):
-            return other.value > other
+            return (self.conv() > other.conv())
     
     def __ge__(self,other):
         if isinstance(other,Distance):
-            return other.value >= other
+            return (self.conv() >= other.conv())
     def __str__(self):
-        "Should return a string representing the distance in meters"
+        return (f"{self.conv()}, is the distance Value in Meter")
```

Comment:
No need for the paren. You can just `return f"{self.conv()}, is the distance Value in Meter"`

Reply:
Oh Great Thanks!